Canes + Cane Blade for Syndicate Librarians

ps3moira opened the pull request:
<!-- Please read these guidelines before opening your PR: https://docs.spacestation14.io/en/getting-started/pr-guideline -->
<!-- The text between the arrows are comments - they will not be visible on your PR. -->

## About the PR
<!-- What did you change in this PR? -->
Added canes to the CuraDrobe. I also added a cane that can be unsheathed to reveal a blade for syndicate librarians. Un Gurde! Have at thee!
## Why / Balance
<!-- Why was it changed? Link any discussions or issues here. Please discuss how this would affect game balance. -->
More syndicate melee weapons are nice, also something for librarians. Maybe someone in the future can make a old trait where if your character is too-old they need a cane? Or just something to be the stations pimp.

## Media
![image](https://github.com/space-wizards/space-station-14/assets/113228053/0e1a1b98-0ae1-4e40-9f97-652d58e3fa89)
![image](https://github.com/space-wizards/space-station-14/assets/113228053/f2fface9-2eb6-483b-ab28-9581cc8d4691)
![Screenshot 2024-03-05 022853](https://github.com/space-wizards/space-station-14/assets/113228053/738b18e3-b868-4e78-a027-3b1dc1d0f052)

- [X] I have added screenshots/videos to this PR showcasing its changes ingame
**Changelog**

:cl:
- add: Added Canes to the CuraDrobe and Cane Blades for Syndicate Librarians

lzk228 commented:
shooting cane for mime?

ps3moira commented:
> shooting cane for mime?

No but I'm thinking on making a shotgun umbrella and umbrellas being a thing.

Hmeister-real commented:
is there a normal cane for normal librarians?

TheShuEd commented:
oh my god, yeah, more secular stuff.

ps3moira commented:
> is there a normal cane for normal librarians?

Normal canes can be found in the CuraDrobe while the Cane Blade can be found in the uplink for librarians.

TheShuEd commented:
test fail is real, fix it

ElPidor commented:
Literraly very garbage item to shit into 20 tc bundle. Up damage to 20 per slash\or give it more reflection, let it apply as hell more bleeding or some special ability _blocking_ etc. I can give my 5 tc to assistant to get my free 6 tc stanbaton AK boxer gloves.  
In 2 words 
Very weak as fuck weapon
**REALLY need buffs**

ps3moira commented:
> Literraly very garbage item to shit into 20 tc bundle. Up damage to 20 per slash\or give it more reflection, let it apply as hell more bleeding or some special ability _blocking_ etc. I can give my 5 tc to assistant to get my free 6 tc stanbaton AK boxer gloves. In 2 words Very weak as fuck weapon **REALLY need buffs**

This weapon is more meant to be a one-on-one stealth weapon not a kill everyone super easily and replace the dual-sword kinda weapon.

SlamBamActionman commented:
> > Literraly very garbage item to shit into 20 tc bundle. Up damage to 20 per slash\or give it more reflection, let it apply as hell more bleeding or some special ability _blocking_ etc. I can give my 5 tc to assistant to get my free 6 tc stanbaton AK boxer gloves. In 2 words Very weak as fuck weapon **REALLY need buffs**
> 
> This weapon is more meant to be a one-on-one stealth weapon not a kill everyone super easily and replace the dual-sword kinda weapon.

I think there is a point that the stats are kinda underwhelming right now. Compare the damage to other stealth/improvised weapons in the uplink: 

- Cybersun Pen: 15 damage, very silent, 1 TC
- Syndie Toolbox: 20 damage, bonk, 2 TC
- Surgical bag: Multiple tools like the saw, 15 damage, 4 TC

My suggestion would be to bump it down in TC and bring it in line with the other options damage-wise. Maybe 17 slash, since it is on par with the pen for concealability but can't edit stamped documents? If you want to make it have some combat flair, you can decrease the damage but give it a faster swing speed to keep the DPS high.

All that being said, cool weapon!

metalgearsloth commented on Resources/Prototypes/Entities/Objects/Weapons/Melee/cane.yml:
```diff
@@ -0,0 +1,101 @@
+        Slash: 12
+    soundHit:
+        path: /Audio/Weapons/bladeslice.ogg
+  - type: Reflect
+    enabled: true
+    reflectProb: .3
+    spread: 90
```
Shouldn't be reflecting.

Hmeister-real replied:
shouldn't it be? Its more tc than an e-dagger, and pretty meh even with the reflect.

metalgearsloth replied:
Energy weapons reflecting is intuitive but random bladed weapons doing it is not.

ps3moira replied:
Fair

ps3moira commented:
I added changes to it's stats. Removed the reflect and buffed the damage 12->14 and attack rate 1.2->1.5. If there's need for more stat changes, I can do so.
